### PR TITLE
[6.5.x] JBPM-5610 - HumanTask ExcludedOwner is able to claim, start and complete task

### PR DESCRIPTION
**Pull request by tsurdilo**

### CODE REVIEW

**Comment by tsurdilo** [comment]
@krisv : is this ok to be merged?

**Comment by psiroky** [comment]
@tsurdilo no, this needs to wait till we finish the 6.4.4 release first.

**Comment by tsurdilo** [comment]
@psiroky ok great. is there a timeline when that will happen ?

**Comment by psiroky** [comment]
@tsurdilo I am not sure about the date, should be soon I guess.

File: jbpm-human-task/jbpm-human-task-core/src/test/java/org/jbpm/services/task/internals/lifecycle/MVELLifeCycleManagerTest.java [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+import org.jbpm.services.task.impl.model.UserImpl;
+import org.junit.Test;
+import org.kie.api.task.model.PeopleAssignments;
+import org.kie.api.task.model.Status;
+import org.kie.api.task.model.Task;
+import org.kie.api.task.model.User;
+import org.kie.internal.task.api.model.InternalTask;
+import org.kie.internal.task.api.model.Operation;
+
+public class MVELLifeCycleManagerTest {
```
Comment (jomarko) [inline jbpm-human-task/jbpm-human-task-core/src/test/java/org/jbpm/services/task/internals/lifecycle/MVELLifeCycleManagerTest.java]: I miss here some negative test, that will assert situation like: ExcludedOwner tries to claim the task.

Comment (tsurdilo) [inline jbpm-human-task/jbpm-human-task-core/src/test/java/org/jbpm/services/task/internals/lifecycle/MVELLifeCycleManagerTest.java]: testClaimIsAllowedExcludedOwner test already does that (it makes sure PermissionDeiniedException is thrown). Are you looking for us to add more tests like that?

Comment (jomarko) [inline jbpm-human-task/jbpm-human-task-core/src/test/java/org/jbpm/services/task/internals/lifecycle/MVELLifeCycleManagerTest.java]: Sorry, I missed the annotation on the test method. Thanks.

**Pull request merged**